src/App.tsx

```diff
@@ -15,9 +17,11 @@ const App: React.FC = () => {
   }
 
   return (
-    <ThemesProvider>
-      <Providers />
-    </ThemesProvider>
+    <I18nextProvider i18n={i18next}>
```

Comment:
is this the correct i18next object, or is this supposed to be from `translations/index.tsx`?